content/docs/next-release-v2/frontend-ui.md

```diff
@@ -71,7 +71,8 @@ An example configuration file (see [complete schema here](https://github.com/jae
     "type": "traces",
     "url": "https://my-logs.server?from=#{startTime | add -60000000 | epoch_micros_to_date_iso}&to=#{endTime | add 60000000 | epoch_micros_to_date_iso}'",
     "text": "Redirect to kibana to view log with formatted dates"
-  }]
+  }],
```

Comment:
could you re-format this json? The `[]` should not be on the same lines with `{}` (e.g. it's not in the `menu:` section).